src/renderer/widgets/CreateWallet/ui/MultisigWallet/FlexibleMultisigWallet.tsx

```diff
@@ -59,33 +59,29 @@ export const MultiChainMultisigWallet = ({ isOpen, onClose, onComplete, onBack }
   }, [isOpen]);
 
   useEffect(() => {
-    createMultisigWalletModel.events.signatoriesChanged(signatories);
-  }, [signatories.length]);
-
-  useEffect(() => {
-    createMultisigWalletModel.events.callbacksChanged({ onComplete });
+    flowModel.events.callbacksChanged({ onComplete });
   }, [onComplete]);
 
-  const goToPrevStep = () => {
-    if (activeStep === Step.INIT) {
-      onBack();
-    } else {
-      setActiveStep((prev) => prev - 1);
-    }
-  };
+  // const goToPrevStep = () => {
```

Comment:
remove it?

Reply:
This is all about the Flexible multisig, which is not implemented in this PR. Hence many things commented, before this is done.